ExaltedApotheosisPreReqs.cs

```diff
@@ -0,0 +1,59 @@
+    //cs_include Scripts/CoreBots.cs
+    //cs_include Scripts/CoreFarms.cs
+    using Skua.Core.Interfaces;
+
+    public class ExaltedApotheosisPreReqs
+    {
+        public IScriptInterface Bot => IScriptInterface.Instance;
+        public CoreBots Core => CoreBots.Instance;
+        public CoreFarms Farm = new();
+
+        public void ScriptMain(IScriptInterface bot)
+        {
+            Core.SetOptions();
+
+            PreReqs();
+
+            Core.SetOptions(false);
+        }
+
+        public void PreReqs()
+        {
+            Core.AddDrop("Exalted Node", "Thaumaturgus Alpha", "Apostate Alpha", "Exalted Relic Piece", "Exalted Forgemetal", "Exalted Artillery Shard");
+            if (Core.CheckInventory("Exalted Node", 300) && Core.CheckInventory("Thaumaturgus Alpha") && Core.CheckInventory("Apostate Alpha"))
+                Core.Logger("Got all prerequisites! Kill the ultra bosses manually for insignias next to complete Exalted Apotheosis.");
+                return;
+
+            if (!Core.CheckInventory("Apostate Alpha") && !Core.CheckInventory("Thaumaturgus Alpha"))
+                Core.EquipClass(ClassType.Solo);
+                while (!Bot.ShouldExit && !Core.CheckInventory("Exalted Node", 300))
+                {

```

Comment:
for a better reading purposes if & While in c# accept single line statement with no curly brackets needed